examples/pos/math.effekt

```diff
@@ -0,0 +1,10 @@
+def main() = {
+    println(cos(0.0));
+    println(sin(0.0));
+    println(log(1.0));
+    println(log1p(0.0));
+    println(exp(0.0));
+    println(pow(2.0, 5.0));
+    println(tan(0.0));
+    println(atan(0.0));
+}
```

Comment:
Nitpick: we tend to use two spaces to format Effekt files. :)

Reply:
Nitpick accepted. Although I did copy the `doubles.effekt` file which uses 4 spaces 🤷